### PR TITLE
Adding thermal conductivity, heat capacity and diffusivity as outputs.

### PR DESCRIPTION
**Pull request by Cristopher-Morales**
## Proposed Changes

This pull request adds the thermal conductivity, heat capacity and diffusivity as output for the incompressible solver in order to help the process of debugging inside SU2. During this process, we found some bugs regarding the diffusivity and thermal conductivity when the FLUID_MIXTURE model is used. Additionally, unused variables for computing the diffusivity have been removed.

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [X] I am submitting my contribution to the develop branch.
- [X] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [X] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

**Comment by bigfooted** [comment]
@Cristopher-Morales What's going on with the multizone test setup?

**Comment by Cristopher-Morales** [comment]
> @Cristopher-Morales What's going on with the multizone test setup?

I have updated the multizone restart file, I will update the residuals of this test case.

File: SU2_CFD/src/output/CFlowIncOutput.cpp [hunk 1]
```diff
@@ -318,6 +318,8 @@ void CFlowIncOutput::SetVolumeOutputFields(CConfig *config){
 
   if (config->GetKind_Solver() == MAIN_SOLVER::INC_RANS || config->GetKind_Solver() == MAIN_SOLVER::INC_NAVIER_STOKES){
     AddVolumeOutput("LAMINAR_VISCOSITY", "Laminar_Viscosity", "PRIMITIVE", "Laminar viscosity");
+    AddVolumeOutput("HEAT_CAPACITY", "Heat_Capacity", "PRIMITIVE", "Heat capacity");
+    AddVolumeOutput("EFFECTIVE_THERMAL_CONDUCTIVITY", "Effective_Thermal_Conductivity", "PRIMITIVE", "Effective thermal conductivity");
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CFlowIncOutput.cpp:322]: For consistency with the custom output variables
```suggestion
    AddVolumeOutput("THERMAL_CONDUCTIVITY", "Effective_Thermal_Conductivity", "PRIMITIVE", "Effective thermal conductivity");
```

File: SU2_CFD/src/output/CFlowOutput.cpp [hunk 2]
```diff
@@ -1159,8 +1159,10 @@ void CFlowOutput::SetVolumeOutputFields_ScalarSolution(const CConfig* config){
   }
 
   if (config->GetKind_Species_Model() != SPECIES_MODEL::NONE) {
-    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++)
+    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++) {
       AddVolumeOutput("SPECIES_" + std::to_string(iVar), "Species_" + std::to_string(iVar), "SOLUTION", "Species_" + std::to_string(iVar) + " mass fraction");
+      AddVolumeOutput("DIFFUSIVITY_" + std::to_string(iVar), "Diffusivity_" + std::to_string(iVar), "PRIMITIVE", "Diffusivity_" + std::to_string(iVar));
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CFlowOutput.cpp:1164]: This function is for solution quantities not for primitives, if you do this here you may break the restarts.
Moving to SetVolumeOutputFields_ScalarResidual should be safer.

Comment (Cristopher-Morales) [reply SU2_CFD/src/output/CFlowOutput.cpp]: Move to SetVolumeOutputFields_ScalarResidual. Thank you for your comment!!

File: SU2_CFD/include/fluid/CFluidScalar.hpp [hunk 3]
```diff
@@ -125,12 +131,12 @@ class CFluidScalar final : public CFluidModel {
   /*!
    * \brief Get fluid thermal conductivity.
    */
-  inline su2double GetThermalConductivity() override { return Kt; }
+  inline su2double GetThermalConductivity() override { return Kt + Mu_Turb * Cp / Prandtl_Number ; }
```
Comment (pcarruscag) [inline SU2_CFD/include/fluid/CFluidScalar.hpp:134]:
```suggestion
  inline su2double GetThermalConductivity() override { return Kt + Mu_Turb * Cp / Prandtl_Number; }
```

File: SU2_CFD/include/variables/CVariable.hpp [hunk 4]
```diff
@@ -1114,6 +1114,12 @@ class CVariable {
    */
   inline virtual su2double GetThermalConductivity(unsigned long iPoint) const { return 0.0; }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] iPoint - Point index.
+   * \return Value of the mass diffusivity.
+   */
+  inline virtual su2double GetDiffusivity(unsigned long iPoint, unsigned short val_ivar) const { return 0.0; }
```
Comment (pcarruscag) [inline SU2_CFD/include/variables/CVariable.hpp:1122]:
```suggestion
  inline virtual su2double GetDiffusivity(unsigned long iPoint, unsigned short val_ivar) const { return 0.0; }

```

File: SU2_CFD/src/output/CFlowOutput.cpp [hunk 5]
```diff
@@ -1190,8 +1191,10 @@ void CFlowOutput::SetVolumeOutputFields_ScalarResidual(const CConfig* config) {
   }
 
   if (config->GetKind_Species_Model() != SPECIES_MODEL::NONE) {
-    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++)
+    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++){
       AddVolumeOutput("RES_SPECIES_" + std::to_string(iVar), "Residual_Species_" + std::to_string(iVar), "RESIDUAL", "Residual of the transported species " + std::to_string(iVar));
+      AddVolumeOutput("DIFFUSIVITY_" + std::to_string(iVar), "Diffusivity_" + std::to_string(iVar), "PRIMITIVE", "Diffusivity_" + std::to_string(iVar));
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CFlowOutput.cpp:1196]:
```suggestion
      AddVolumeOutput("DIFFUSIVITY_" + std::to_string(iVar), "Diffusivity_" + std::to_string(iVar), "PRIMITIVE", "Diffusivity of the transported species " + std::to_string(iVar));
```

File: SU2_CFD/src/output/CFlowOutput.cpp [hunk 6]
```diff
@@ -1324,6 +1327,7 @@ void CFlowOutput::LoadVolumeData_Scalar(const CConfig* config, const CSolver* co
     for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++) {
       SetVolumeOutputValue("SPECIES_" + std::to_string(iVar), iPoint, Node_Species->GetSolution(iPoint, iVar));
       SetVolumeOutputValue("RES_SPECIES_" + std::to_string(iVar), iPoint, solver[SPECIES_SOL]->LinSysRes(iPoint, iVar));
+      SetVolumeOutputValue("DIFFUSIVITY_"+ std::to_string(iVar), iPoint,Node_Species->GetDiffusivity(iPoint,iVar));
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CFlowOutput.cpp:1330]:
```suggestion
      SetVolumeOutputValue("DIFFUSIVITY_"+ std::to_string(iVar), iPoint, Node_Species->GetDiffusivity(iPoint,iVar));
```

**Review by pcarruscag — APPROVED** [review]
LGTM

**Pull request merged**